src/public/js/pipe-web-monetization.js

```diff
@@ -37,105 +36,127 @@ class Batcher {
       return;
     }
     const totalValue = this.transactionsList.reduce(
-      (previousValue, currentValue) => previousValue + currentValue.value, 0
+      (previousValue, currentValue) => previousValue + currentValue.value,
+      0
     );
     const data = {
-      "pluginId": this.pluginId,
-      "paymentPointer": this.paymentPointer,
-      "date": new Date().getTime(),
-      "totalValue": totalValue,
-      "transactions": this.transactionsList
-    }
-    await fetch(`https://94w4fmrdq3.execute-api.us-east-1.amazonaws.com/Dev/api/transactions`, {
-      method: 'POST',
-      body: JSON.stringify(data),
-      headers: 
+      pluginId: this.pluginId,
+      paymentPointer: this.paymentPointer,
+      date: new Date().getTime(),
+      totalValue: totalValue,
+      transactions: this.transactionsList,
+    };
+    await fetch(
+      `https://94w4fmrdq3.execute-api.us-east-1.amazonaws.com/Dev/api/transactions`,
       {
-        'Content-Type': 'application/json',
+        method: "POST",
+        body: JSON.stringify(data),
+        headers: {
+          "Content-Type": "application/json",
+        },
       }
-    });
+    );
     this.transactionsList = [];
   }
 }
 
 jQuery(function ($) {
-  
   function setupMetaTag(pointer) {
     $("head").append(pointer);
   }
 
   if (!ajax_variables.logged_in) {
-    const pointer = "<meta name='monetization' content='" + $('#monetization').attr('name') + "' />";
-    setupMetaTag(pointer)
-    setupMonetizationListeners()
+    const pointer =
+      "<meta name='monetization' content='" +
+      $("#monetization").attr("name") +
+      "' />";
+    setupMetaTag(pointer);
+    setupMonetizationListeners();
   }
 
-
   async function setupMonetizationListeners() {
     const batcher = new Batcher();
 
-      if (!document.monetization) {
-        return;
-      }
+    if (!document.monetization) {
+      return;
+    }
+
+    batcher.setPluginId(plugin_options.pwm_plugin_id);
+    batcher.setPaymentPointer(
+      document
+        .querySelector('meta[name="monetization"]')
+        .getAttribute("content")
+    );
 
-      batcher.setPluginId(plugin_options.pwm_plugin_id);
-      batcher.setPaymentPointer(document.querySelector('meta[name="monetization"]').getAttribute('content'));
-
-      document.monetization.addEventListener(
-        'monetizationprogress',
-        (event) => {
-          batcher.add({
-            date: new Date().getTime(),
-            value: Number((Number(event.detail.amount) * (10**(-1*event.detail.assetScale))).toFixed(event.detail.assetScale))
-          });
-        }
-      );
-  
-      batcher.scheduleFlush();
+    document.monetization.addEventListener("monetizationprogress", (event) => {
+      batcher.add({
+        date: new Date().getTime(),
+        value: Number(
+          (
+            Number(event.detail.amount) *
```

Comment:
This adjustment in the value might be introducing uncertaintiy to the data?

Reply:
I don't think so, here it worked normal. It's just an auto indent